dockerfiles/ubuntu/is/init.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
```

Comment:
@chamindi-a AFAIU you have added the incorrect Docker container entry point script to WSO2 Identity Server Docker image resources. Please correct this.

Reply:
This is fixed in https://github.com/wso2/docker-is/pull/71/commits/0aed5ecaa0ef6bec89bc808c3008bc71dbc6e653.